### PR TITLE
Build/create docker build image

### PR DESCRIPTION
**Pull request by lukasholzer**
### <strong>Pull Request</strong>

Create a workspace build image that can be used for faster CI build times on Jenkins for barista.

In a follow up it can be used on the circleci to skip installing on every build, and use our custom image for building like:

```yml
  build-test:
    working_directory: /dynatrace
    docker:
      - image: designops/workspace-base:latest
    steps:
      - attach_workspace:
          at: /dynatrace
      - run: ./node_modules/.bin/nx workspace-lint
      - run: yarn affected:lint --all --parallel
```

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,5 +1,20 @@
-node_modules
-.git
-.github
-.circleci
-.vscode
+#--------------------------
```
Comment (tomheller) [inline .dockerignore]: this change might be problematic for these github actions, like this one: 
https://github.com/dynatrace-oss/barista/blob/master/.github/workflows/cherry-picker.yml
https://github.com/dynatrace-oss/barista/tree/master/tools/cherry-picker

Can we check if they work first?

Comment (lukasholzer) [reply .dockerignore]: they have their own `.dockerignore` files. So I thought it should be safe but how can we test them?

Comment (lukasholzer) [reply .dockerignore]: I will add some merge-ready label and wait if the pull request labeller is working. 

Consider the label `merge-ready` as **not merge ready** :D

Comment (lukasholzer) [reply .dockerignore]: image

The pull request labeller worked fine and it has the same configuration as the cherry-picked.  So everything is working! 👍

**Pull request merged**